src/core/core.cpp

```diff
@@ -399,15 +395,6 @@ bool Core::writeScreen(QString& fileName, QString& format, bool tmpScreen)
     if (!fileName.contains(QStringLiteral(".") + format))
         fileName.append(QStringLiteral(".") + format);
 
-    // saving temp file (for uploader module)
-    if (tmpScreen)
-    {
-        if (!fileName.isEmpty())
-            return _pixelMap->save(fileName, format.toLatin1().constData(), _conf->getImageQuality());
-        else
-            return false;
-    }
-
```

Comment:
This block shouldn't be removed; it isn't limited to upload but is also used on double clicking the screenshot area.

You could remove "(for uploader module)" instead.